### ISSUE TITLE
nodeSelector not in schema

### ISSUE THREAD

**Issue by swapkh91** [linsoss/doris-operator#15]
I'm trying to schedule FE, BE, broker on separate nodepools on GKE cluster, following the method shown [here](https://linsoss.github.io/doris-operator/docs/deploy/configure-doris-cluster/#use-nodeselector-to-schedule-pods)

When I run `kubectl apply`, I get the following error
```
Error from server (BadRequest): error when creating "doris-cluster.yaml": DorisCluster in version "v1beta1" cannot be handled as a DorisCluster: strict decoding error: unknown field "spec.be.nodeSelector", unknown field "spec.cn.nodeSelector", unknown field "spec.fe.nodeSelector"
```

doris-cluster.yaml

```
apiVersion: al-assad.github.io/v1beta1
kind: DorisCluster
metadata:
  name: basic
spec:
  # Image tag of fe, be, cn and broker components.
  version: 2.0.2
  ###################
  # FE Configuration #
  ###################
  fe:
    baseImage: ghcr.io/linsoss/doris-fe

    nodeSelector:
      cloud.google.com/gke-nodepool: fe-pool

    ## The replica of fe must be an odd number, it is recommended to 3 in the production env.
    replicas: 1

    ## Extra FE config, see: https://doris.apache.org/docs/dev/admin-manual/config/fe-config/
    config:
      prefer_compute_node_for_external_table: 'true'
    #   qe_max_connection: '2048'

    requests:
      cpu: '5'
      memory: 8Gi
      storage: 50Gi
    limits:
      cpu: '5'
      memory: 8Gi

  ###############################
  # BE(mixed mode) Configuration #
  ###############################

  be:
    ## Base image of the BE component
    baseImage: ghcr.io/linsoss/doris-be

    nodeSelector:
      cloud.google.com/gke-nodepool: be-pool

    ## The replica of the BE component
    replicas: 1

    ## Extra BE config, see: https://doris.apache.org/docs/dev/admin-manual/config/be-config
    config:
     external_table_connect_timeout_sec: '30 seconds'
    #  mem_limit: '90%'

    requests:
      cpu: '5'
      memory: 25Gi
      storage: 90Gi
    limits:
      cpu: '5'
      memory: 25Gi

  #####################################
  # BE(computation mode) Configuration #
  #####################################

  cn:
    ## Base image of the CN component
    baseImage: ghcr.io/linsoss/doris-cn

    nodeSelector:
      cloud.google.com/gke-nodepool: cn-pool

    replicas: 1

    ## Extra BE config, see: https://doris.apache.org/docs/dev/admin-manual/config/be-config
    config:
      external_table_connect_timeout_sec: '30 seconds'
      mem_limit: '90%'

    requests:
      cpu: '5'
      memory: 25Gi
    limits:
      cpu: '5'
      memory: 25Gi

  #######################
  # Broker Configuration #
  #######################

  broker:
    ## Base image of the Broker component
    baseImage: ghcr.io/linsoss/doris-broker

    ## The replica of the Broker component
    replicas: 1

    ## Extra Broker config.
    config: { }

    requests:
      cpu: '1'
      memory: 2Gi
    #    limits:
    #      cpu: 8
    #      memory: 16Gi
```

I installed the operator as follows:
```
mkdir doris-operator
flux pull artifact oci://ghcr.io/linsoss/kustomize/doris-operator:1.0.0 --output ./doris-operator/
kubectl apply -k doris-operator
```

**Comment by Al-assad** [comment]
@swapkh91 Hi Khare, `.spec.be` does not currently support the configuration of the `nodeSelector` property. You can skip this configuration item for now, I'll patch this feature in the next minor release soon 😃.

**Comment by swapkh91** [comment]
> @swapkh91 Hi Khare, `.spec.be` does not currently support the configuration of the `nodeSelector` property. You can skip this configuration item for now, I'll patch this feature in the next minor release soon 😃.

@Al-assad got it, thanks for the reply

**Comment by Al-assad** [comment]
@swapkh91  Hi Khare, I just released Doris-Operator [v1.0.1](https://github.com/linsoss/doris-operator/releases/tag/v1.0.1) which provides support for NodeSelector. You can update Doris-Operator by following the steps below:
```
flux pull artifact oci://ghcr.io/linsoss/kustomize/doris-operator:1.0.1 --output ./doris-operator/
kubectl apply -k doris-operator
```

**Comment by swapkh91** [comment]
> @swapkh91 Hi Khare, I just released Doris-Operator [v1.0.1](https://github.com/linsoss/doris-operator/releases/tag/v1.0.1) which provides support for NodeSelector. You can update Doris-Operator by following the steps below:
> 
> ```
> flux pull artifact oci://ghcr.io/linsoss/kustomize/doris-operator:1.0.1 --output ./doris-operator/
> kubectl apply -k doris-operator
> ```

@Al-assad thanks, works now
